### PR TITLE
Build work-section

### PR DESCRIPTION
**Pull request by Elijahdre**
In this project, I;
- Created the work and about section of my portfolio using HTML and CSS codes.
- Added index.html and style.css files for the HTML and CSS codes respectively.
- Added linters file to correct errors in the codes.
- Downloaded some images from Figma my project's design.
- Added a descriptive README.md file.

### CODE REVIEW

**Review by chrystalme — CHANGES_REQUESTED** [review]
## Hi @Elijahdre , 

You have done a good job so far!
However, there are some issues that you still need to work on to prepare your project for further evaluation
## Highlights 💡 
✔️ Linters well set up and passes all tests
✔️ Nice PR description and summary of what it contains
✔️ Nice code and use of semantic HTML 
## Suggested changes ♻️ 
_Check the comments under the review._
Your Design seems to be a little off
|Figma Design |Your Design|
|---|---|
|image|image|

- There is no difference in the background color from the `header section` to the just before the `footer`.
- I observed that you have just one (1) commit to this branch. Kindly make frequent commits to avoid one thinking that you copied your code from somewhere else.



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA). Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -116,3 +116,348 @@
...
+  display: grid;
+  grid-template-columns: auto;
+  gap: 10px;
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  margin: 50px auto;
+  padding: 16px 0;
+  position: static;
+}
```
Comment (HAMMAS-SALEEM) [inline style.css:138]: - [ ] As the previous reviewer said that you should make it display: flex. Please make the styling according to the image below :+1:
  
inner-container-flex

File: style.css [hunk 2]
```diff
@@ -116,3 +116,348 @@
...
   align-items: baseline;
 }
+
+#work-container {
+  display: grid;
+  flex-direction: column;
+  gap: 20px;
+  padding-top: 12px;
+  background-color: #fff;
+}
```
Comment (HAMMAS-SALEEM) [inline style.css:126]: - [ ] As the previous reviewer requested you need to change the background. It's good that you have changed the background :+1:. But It's not visible. :cry: Kindly change the background color according to the instructions

background-color-fix

File: style.css [hunk 3]
```diff
@@ -116,3 +116,348 @@
...
+  font: normal normal 400 16px 'Poppins', sans-serif;
+  position: static;
+  width: 370px;
+  height: 144px;
+  left: 0;
+  top: 113px;
+  line-height: 24px;
+  color: #344563;
+  margin: 12px 0;
+}
```
Comment (HAMMAS-SALEEM) [inline style.css:352]: - [ ] You have a horizontal scrollbar. Please fix the styling here as in the image. :+1:
horizontal-scroll-fixed

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Changes Requested :recycle:

almost there

## Hey @Elijahdre  :wave:
### Highlights :1st_place_medal:

Your project is working :clap:
Good commit history :heavy_check_mark:
Following Github Flow :heavy_check_mark:
Following HTML, CSS best practices :heavy_check_mark:
Using .gitignore file :heavy_check_mark:
Meaningful commit messages :heavy_check_mark:

You have made a great effort implementing some of the changes requested by the previous reviewer in your project :blush: and I can see that but you need to work on some issues to go to the next project but you are almost there. :+1:

### **_Any requested change with the optional tag is not crucial enough to prevent you from getting approval. But we, highly recommend that you to take these changes into consideration_**

### Once you implement all the changes you should be ready to submit your project for a re-review :rocket: :rocket:

Cheers and Happy coding!👏👏👏

### Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. You can reach out to me on slack by searching my name (Hammas Bin Saleem) for any explanation.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -116,3 +116,365 @@
...
 }
+
+/* #work-container {
+  display: grid;
+  flex-direction: column;
+  gap: 20px;
+  padding-top: 12px;
+  background-color: #fff;
+} */
+
```
Comment (tresorsawasawa) [inline style.css:127]: - [ **OPTIONAL** ] **IMPORTANT:** Kindly make sure to delete all the commented code lines to make your code base cleaner and professional.

File: style.css [hunk 5]
```diff
@@ -116,3 +116,365 @@
...
+  display: grid;
+  grid-template-columns: auto;
+  gap: 10px;
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  margin: 50px auto;
+  padding: 16px 0;
+  position: static;
+} */
```
Comment (tresorsawasawa) [inline style.css:146]: - [ **OPTIONAL** ] **IMPORTANT:** Kindly make sure to delete all the commented code lines to make your code base cleaner and professional.

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Elijahdre 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -77,6 +77,207 @@ <h1 class="hero-h1">
           </ul>
         </div>
       </section>
+      <section id="work-container">
+        <div class="container">
```
[the remark this answers is not in the document]
Comment (Elijahdre) [reply index.html:81]: @chrystalme Thank you for the review. I will do that now.

File: style.css [hunk 7]
```diff
@@ -116,3 +116,348 @@
...
+  display: grid;
+  grid-template-columns: auto;
+  gap: 10px;
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  margin: 50px auto;
+  padding: 16px 0;
+  position: static;
+}
```
[the remark this answers is not in the document]
Comment (Elijahdre) [reply style.css]: @chrystalme I will make it to display: flex now. Thank you🙏

File: style.css (hunk 1 above)
Comment (Elijahdre) [reply style.css]: @HAMMAS-SALEEM Thank you very much for this.

File: style.css (hunk 2 above)
Comment (Elijahdre) [reply style.css]: @HAMMAS-SALEEM Thank you for this thorough review. I will fix it now.

File: style.css (hunk 3 above)
Comment (Elijahdre) [reply style.css]: @HAMMAS-SALEEM Ok got it! Thank you🙏

File: style.css (hunk 4 above)
Comment (Elijahdre) [reply style.css:127]: @tresorsawasawa Alright! I will make sure of that. Thank you🙏

File: style.css (hunk 5 above)
Comment (Elijahdre) [reply style.css:146]: @tresorsawasawa Noted! Thank you.

**Pull request merged**